notice.go

```diff
@@ -36,13 +37,22 @@ func getDefaultContext() map[string]interface{} {
 			defaultContext["rootDirectory"] = wd
 		}
 
-		if s := os.Getenv("GOPATH"); s != "" {
+		if s := gopath(); s != "" {
 			defaultContext["gopath"] = s
 		}
 	})
 	return defaultContext
 }
 
+func gopath() string {
+	path := os.Getenv("GOPATH")
+	if path != "" {
+		return path
+	}
+
+	return filepath.Join(os.Getenv("HOME"), "go")
```

Comment:
Add a check that `$HOME` is defined. It can be missing on platforms like Windows.